File: src/main/java/com/cartobucket/auth/routes/mappers/ClientRequestFilterMapper.java
package com.cartobucket.auth.routes.mappers;

import com.cartobucket.auth.model.generated.ClientRequestFilter;

import java.util.List;
import java.util.UUID;

public class ClientRequestFilterMapper {
    public static ClientRequestFilter to(List<UUID> authorizationServerIds) {
        var filter = new ClientRequestFilter();
        filter.setAuthorizationServerIds(authorizationServerIds);
        return filter;
    }
}

File: auth-api/src/main/kotlin/com/cartobucket/auth/api/server/routes/mappers/ClientRequestFilterMapper.kt
// (C)2024
package com.cartobucket.auth.api.server.routes.mappers

import com.cartobucket.auth.api.dto.ClientRequestFilter
import java.util.UUID

object ClientRequestFilterMapper {
    fun to(authorizationServerIds: List<UUID>): ClientRequestFilter =
        ClientRequestFilter(
            authorizationServerIds = authorizationServerIds,
        )
}
